<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Index :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="indexLineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Find Text:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="findTextlineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="captureValueBtn">
         <property name="text">
          <string>Capture Value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setValueBtn">
         <property name="text">
          <string>Set Value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="getValueBtn">
         <property name="text">
          <string>Get Values</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="findTextBtn">
         <property name="text">
          <string>Find Text</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeItemBtn">
         <property name="text">
          <string>Remove Item</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
